test/functional/apps/discover/_doc_table.ts

```diff
@@ -59,6 +60,46 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.timePicker.setDefaultAbsoluteRange();
     });
 
+    describe('classic table in window 900x700', async function () {
```

Comment:
Note that in our functionals we seem to have a very little number of tests for little screen sizes